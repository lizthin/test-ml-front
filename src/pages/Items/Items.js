import React from 'react';
import ItemsRouter from './routes/ItemsRouter';

/**
 * funtion component Items, container route items
 * @param {props object} props history props route  
 * @returns 
 */
const Items = ({history}) => <ItemsRouter parentHistory={history}/>
export default Items;