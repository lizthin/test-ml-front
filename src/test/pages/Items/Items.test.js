import React from 'react';
import { shallow } from 'enzyme';
import Items from '../../../pages/Items/Items';


describe('Test Items', () => {
    test('should display component <ItemsRouter history={mockProperties} />', () => {
        const mockProperties = {
            location: {
                hash: "",
                key: "6ti0yy",
                pathname: "/items",
                search: "",
            },
            push: jest.fn()
        };
        const wrapper = shallow(<Items history={mockProperties} />);
        expect(wrapper).toMatchSnapshot();
    })
});