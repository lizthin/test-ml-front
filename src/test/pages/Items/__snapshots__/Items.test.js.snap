// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Items should display component <ItemsRouter history={mockProperties} /> 1`] = `
<ItemsRouter
  parentHistory={
    Object {
      "location": Object {
        "hash": "",
        "key": "6ti0yy",
        "pathname": "/items",
        "search": "",
      },
      "push": [MockFunction],
    }
  }
/>
`;
